src/app/util/ember-compatibility-functions.cpp

```diff
@@ -544,6 +547,7 @@ CHIP_ERROR prepareWriteData(const EmberAfAttributeMetadata * metadata, TLV::TLVR
 }
 } // namespace
 
+#if 0
```

Comment:
This is commentred out code - is this intended?

General rule for commented out code would be to not check it in but rather delete it (it can always be recovered using git history)

Reply:
Yes error, I did this robotically in baby steps, that function is gone now (merged into WriteSingleClusterData, no reason to keep it separate), I will fix.

Reply:
Fixed now.